atqa-framework/tests/e2e/features/ChromeUpdate.feature

```diff
@@ -0,0 +1,219 @@
+    And I populate seed type 'TESTNET_SEED_1' with prefix 'CUSTOM_SEED'
+    Then The Finish button is 'available' on the Import page
+    And I see 'Your accounts are ready to use' message
+    And I see 'You can now close this tab and continue using the extension or add more accounts.' message
+    When I on the 'CHROMIUM_EXTENSIONS' page
+    And I click on the Extension details button in Chrome
+    Then I see Keeper version in Chrome is 'not equal' to latest
+    And I initialize 'chrome_dir_update' folder and update extension on the latest version
+    And I click on the Update Chromium extension button
+    And I restart Chrome extension
+    Then I see Keeper version in Chrome is 'equal' to latest
+    When I on the 'POPUP' page for id 'chromeId'
+    And I fill default password
+    And I submit default password
+    Then The latest Keeper version 'available' in the modal page
+    And The account 'CUSTOM_SEED' is 'available' on the asset tab
+    When I click on the Choose Account button
+    And I choose account at position '1'
+    Then The account 'TESTNET_SEED' is 'available' on the asset tab
+    When I click on the 'Assets' tab
+    Then I see WAVES asset is 'available' on the asset page
+    And The assets of embedded Testnet seed is 'available' on the Assets tab
+    When I click on the Choose Account button
+    And I choose account at position '2'
+    Then The account 'TESTNET_EMAIL' is 'available' on the asset tab
+    When I click on the 'Assets' tab
+    Then The assets of embedded Testnet email is 'available' on the Assets tab
+    When I choose 'Stagenet' network
+    Then The account 'STAGENET_SEED' is 'available' on the asset tab
+    When I click on the 'Assets' tab
+    Then The assets of embedded Stagenet seed is 'available' on the Assets tab
+    When I choose 'Mainnet' network
+    Then The account 'MAINNET_SEED' is 'available' on the asset tab
+    When I click on the 'Assets' tab
+    Then I see WAVES asset is 'available' on the asset page
+    And The assets of embedded Mainnet seed is 'available' on the Assets tab
+    When I click on the Choose Account button
+    And I choose account at position '1'
+    Then The account 'MAINNET_EMAIL' is 'available' on the asset tab
+    And I see WAVES asset is 'available' on the asset page
+    When I click on the 'NFTs' tab
+    Then I see 'You don't have any NFT yet' message
+    When I click on the 'History' tab
+    Then I see 'There's nothing to show yet' message
+    When I on the 'CHROMIUM_EXTENSIONS' page
+    And I click on the Extension details button in Chrome
+    Then I see Keeper version in Chrome is 'equal' to latest
+    Given I initialize 'chrome_dir_init' folder and update extension on the previous version
+
+  Scenario: Update Keeper with active transaction
+    When I click on the 'Assets' tab
+    And I click on the More button
+    And I click on the Send asset button
+    And I fill amount '10' for Embedded Stagenet seed
```

Comment:
I think this should be split into two steps, one for filling an amount and one for filling an address